stylist/api/views.py

```diff
@@ -41,7 +41,7 @@ def perform_create(self, serializer):
                site = Site.objects.get_current()
           new_name = previous.name + " copy"
           instance = serializer.save(site=site, attrs=previous.attrs, name=new_name)
-          if not getattr(settings, 'STYLIST_IGNORE_SASS', False):
+          if not getattr(settings, 'STYLIST_IGNORE_SASS', True):
```

Comment:
This would be a bit cleaner with the other naming convention and with out so many "nots"